src/django_management/settings.py

```diff
@@ -38,6 +39,11 @@
     'django.contrib.staticfiles',
     #other apps
     'accounts',
+    'django.contrib.sites',
```

Comment:
for these - could you add a comment to separate out what these are for ? 

```python
     # google login providers
     'django.contrib.sites',
     'allauth',
     'allauth.account',
     'allauth.socialaccount',
     'allauth.socialaccount.providers.google',
     #other apps
     'accounts',
```

Reply:
These are apps required for social auth. This is like a template that we get in Django's initial setup.

Reply:
I think it'd be nice to include a comment saying that these are for social auth just to help keep things organized and explain what the different `installed_apps` are for

Reply:
Done